src/main/controllers/ListTypeController.ts

```diff
@@ -0,0 +1,32 @@
+import {PipRequest} from '../models/request/PipRequest';
+import { Response } from 'express';
+import {cloneDeep} from 'lodash';
+import moment from 'moment';
+import { PublicationService } from '../service/publicationService';
+
+const publicationService = new PublicationService();
+
+export default class ListTypeController {
```

Comment:
this looks more like a SJP public list controller, which would then be called from the SOP page when the file that is a SJP public list is clicked